src/java/org/apache/cassandra/streaming/StreamLockfile.java

```diff
@@ -85,7 +88,7 @@ this class will not need to clean up tmp files (on restart), CassandraDaemon doe
         }
         catch (IOException e)
         {
-            logger.warn(String.format("Could not create lockfile %s for stream session, nothing to worry too much about", lockfile), e);
+            logger.warn("Could not create lockfile {} for stream session, nothing to worry too much about", SafeArg.of("lockfile", lockfile.toString()), e);
```

Comment:
file marked as safe, inconsistent with the other files that i believe we've decided are unsafe